pkg/api/applications/applications.go

```diff
@@ -47,18 +48,20 @@ func Init(listers controller.Listers) {
 }
 
 func refreshApplications() error {
+	// Create log instance for refreshing applications
+	logger := zerolog.New(os.Stderr).With().Timestamp().Str("kind", "Applications").Str("name", "Refresh").Logger()
```

Comment:
field "kind", "Applications" and "name", "Refresh" should be removed. I've commented elsewhere about fields as well. We should go through and remove all these types of fields until we've decided what they should be